Code-the-Dream-School/ee-prac-team3-front
# Improved user correctness feedback

## Purpose:
To improve user experience when receiving correctness feedback from application

## Summary:
Updated front end to:
-creates a state array in QuizContent component to track user answer selections
-Conditionally colors box and outline of chosen user radio answer to green if user answer is correct and red if user answer is incorrect.
-creates a message box for correct and incorrect answer choices.
-conditionally renders further resources if "resources" key in question data has a value
-cleaned up consoles and added comments

## Front End Change:
Main.jsx
QuizContent.jsx
QuestionContent.jsx
exportFunctions.js

## Test Steps:
User can take a quiz and see if conditional render works on each question.  User can then navigate to answered questions and should still be able to see the feedback in the UI.
